src/store/reducers/pdisk/utils.ts

```diff
@@ -34,18 +35,20 @@ export function preparePDiskDataResponse([pdiskResponse = {}, nodeResponse]: [
         TotalSize: PDiskTotalSize,
         SystemSize,
         ExpectedSlotCount,
-        EnforcedDynamicSlotSize,
+        SlotSize,
     } = preparedPDisk;
 
     let logSlot: SlotItem<'log'> | undefined;
 
     if (valueIsDefined(LogTotalSize)) {
+        const usagePercent = (Number(LogUsedSize) * 100) / Number(LogTotalSize);
+
         logSlot = {
             SlotType: 'log',
             Used: Number(LogUsedSize),
             Total: Number(LogTotalSize),
-            UsagePercent: (Number(LogUsedSize) * 100) / Number(LogTotalSize),
-            Severity: 1,
+            UsagePercent: usagePercent,
+            Severity: getSpaceSeverity(usagePercent),
```

Comment:
Colorize log just like other slots. For example, if log is 90/100, it should be yellow, if 99/100 - red.